modules/kernel/mkosi.build.chroot

```diff
@@ -87,6 +91,13 @@ export KBUILD_BUILD_USER="mkosi"
 
 make $EXTRA -j "$(nproc)" $( ((INCREMENTAL)) && echo modules)
 
+# We put the compile_commands.json in /work/build
+# because we do not have access to the kernel-specific build directory
+# at the top of the file when starting clangd.
+# Postponing the execution of clangd until we know the proper build directory
+# seems to mess with the language server initialization.
+scripts/clang-tools/gen_compile_commands.py -d "$BUILDDIR" -o /work/build/compile_commands.json
+
```

Comment:
This is kinda annoying when working with different kernel versions but reusing the same build dir (which I am, as I just `Include=` this repo) as the latest compilation database wins. I tried to remove this and put the clangd invocation below but in that case the VSCode extension did not finish initializing. It might work if we redirect all of preceding output to stdout into devnull but that would hinder other usage.